app/components/common/Paragraph.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { View, StyleSheet, Dimensions } from 'react-native'
+import RNText from './RNText'
+import colors from '../../config/colors'
+
+const windowWidth = Dimensions.get('window').width
+const Paragraph = ({ title, content }) => {
```

Comment:
I think naming here is too generic? Maybe change the name to DetailSlice or something?